### ISSUE TITLE
TEE: exposing rotation angle of sampling grid

### ISSUE THREAD

**Issue by IngmarVoigt2** [MedicalUltrasound/Image3dAPI#160]
Dear @forderud  @SteveKauffman,

are there any plans for exposing the rotation angle of the sampling grid, which is a user defined parameter for matrix TEE probes? Generally it is very useful to know this value, in order to inversely rotate the image to obtain consistent anatomical orientation

Just in case I might  not be using the right terminology, please see below for some example screenshots from some DICOM thumbnails

# GE
![image](https://user-images.githubusercontent.com/52462518/75816527-abad6800-5d95-11ea-8ede-0daba8056289.png)

# Philips
![image](https://user-images.githubusercontent.com/52462518/75816633-da2b4300-5d95-11ea-8b53-baa82478023a.png)

**Comment by IngmarVoigt2** [comment]
Also, one more question in terms of usage: if I would manipulate the bounding box to arbitrary axes, would the volume be resampled accroding to this (modified) coordinate system (since it's an input parameter according to https://github.com/MedicalUltrasound/Image3dAPI/blob/master/Image3dAPI/IImage3d.idl#L288)?

**Comment by forderud** [comment]
> Also, one more question in terms of usage: if I would manipulate the bounding box to arbitrary axes, would the volume be resampled accroding to this (modified) coordinate system (since it's an input parameter according to https://github.com/MedicalUltrasound/Image3dAPI/blob/master/Image3dAPI/IImage3d.idl#L288)?

Yes, conforming loaders should support arbitrary bounding-box orientation.

**Comment by IngmarVoigt2** [comment]
Thanks @forderud! Any feedback on the rotation angle of sampling grid? Is there a possibility to read that number from the DICOM directly?

@SteveKauffman how would I read this from Philips DICOMs?

**Comment by IngmarVoigt2** [comment]
In addition some of the Philips data shows X-Plane thumbnails (and according transducer rotation, see below), but actually has 3D data inside, which then makes it hard to guess - so ideally all this could be read from the API

![image](https://user-images.githubusercontent.com/52462518/93907448-a138c880-fcfd-11ea-8fa3-f03b5500ea06.png)
